examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -118,6 +118,77 @@ persistentvolumeclaim "claim1" deleted
 $ kubectl get pv
 
 ```
+### Provision Ceph RBD Volume
+Enable RBD provisioning by supplying `--enable-network-storage-provisioner=true --storage-config=/path/to/storage/config/directory` to `kube-controller-manager`.
+
+In the storage configuration directory, provide a ceph cluster configuration json like the following. Note, only `Mon`, `Pool`, `Id`, and `Keyring` are mandatory.
+```json
+{
+  "Mon": ["10.16.154.78:6789"],
+  "Pool": "kube",
+  "Id": "kube",
+  "Keyring": "/etc/ceph/ceph.client.kube.keyring",
+  "FSType": "ext4"
+}
+```
+
+Create a PV json like the following:
+
+```json
+{
+  "kind": "PersistentVolumeClaim",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "claim1",
+    "annotations": {
+      "volume.alpha.kubernetes.io/storage-class": "foo"
+    }
+  },
+  "spec": {
+    "accessModes": [
+      "ReadWriteOnce"
+    ],
+    "resources": {
+      "requests": {
+        "storage": "1Mi"
+      }
+    }
+  }
+}
+```
+
+Create the PV using `kubectl`:
+
+```console
+# kubectl create -f claim1.json 
+persistentvolumeclaim "claim1" created
+```
+
+Confirm the PV is created using RBD volume:
+```console
+# kubectl.sh get pv
+NAME           LABELS    CAPACITY   ACCESSMODES   STATUS    CLAIM            REASON    AGE
+pv-rbd-7oiyw   <none>    1Mi        RWO           Bound     default/claim1             12s
```

Comment:
Given that namespace cannot contain dots but PV names can, couldn't we name the generated PV "{{.PVC.Namespace}}.{{.PVC.Name}}"?